test/pattern.spec.ts

```diff
@@ -0,0 +1,62 @@
+import execa from 'execa'
+import mock from 'mock-fs'
+
+import prettyQuick from 'pretty-quick'
+
+jest.mock('execa')
+
+afterEach(() => {
+  mock.restore()
+  jest.clearAllMocks()
+})
+
+describe('match pattern', () => {
+  it('issue #73 #125 - regex grouping (round pattern)', () => {
+    const onFoundChangedFiles = jest.fn()
+
+    mock({
+      '/.git': {},
+      '/src/apps/hello/foo.js': "export const foo = 'foo'",
+      '/src/libs/hello/bar.js': "export const bar = 'bar'",
+      '/src/tools/hello/baz.js': "export const baz = 'baz'",
+      '/src/should-not-be-included/hello/zoo.js': "export const zoo = 'zoo'",
+    })
+
+    // @ts-expect-error -- Need to find a better way to moch this
```

Comment:
```suggestion
    // @ts-expect-error -- Need to find a better way to mock this
```

Oh, sorry, please help to fix all the typos 🤣